src/type-translator.ts

```diff
@@ -270,6 +270,9 @@ export class TypeTranslator {
         typeStr += this.translate(referenceType.target);
       }
       if (referenceType.typeArguments) {
+        // Closure doesn't accept a type like {?<...>}, so if the parameterized
+        // type itself is unknown just use that by itself.
```

Comment:
This came up in an obscure Angular2 case that I wasn't able to reproduce in a test.  Something about a type-parameterized interface used in destructuring where the interface itself failed to convert to a Closure type...